earthpy/spatial.py

```diff
@@ -392,6 +392,91 @@ def crop_image(raster, geoms, all_touched=True):
     return out_image, out_meta
 
 
+def crop_all(
+    raster_paths,
+    output_dir,
+    geoms,
+    overwrite=False,
+    all_touched=True,
+    verbose=True,
+):
+    """Takes a list of rasters and a boundary, and crops them efficiently.
+
+    Parameters
+    ----------
+    raster_paths : list of file paths
+        List of paths of rasters that will be cropped.
+    output_dir : list or string
```

Comment:
@nkorinek  this  might need to be updated? just takes a string now?

Reply:
@lwasser fixed that!